$NetBSD: patch-ac,v 1.1 1999/10/04 21:25:14 hubertf Exp $

This fixes an error that will only occur on machines fast enough to
compile through two directories in the "libray" directorie in 1 
second: the upto-date check of libray.a will fail because it was already
touched in this very second. Slowing things down here will help.
 
(I seem to remember some hacking about this before, maybe in make(1),
but I don't remember, and I won't start hacking make(1) now ;-)


--- libray/Makefile.orig	Mon Oct  4 23:12:55 1999
+++ libray/Makefile	Mon Oct  4 23:13:10 1999
@@ -10,7 +10,7 @@
 
 default:
 	for i in $(STUFF); do \
-		(cd $$i && $(MAKE)); \
+		(echo $$i ; cd $$i && $(MAKE) ; sleep 1); \
 	done
 
 #
